src/components/deals/details/DealDetails.tsx

```diff
@@ -0,0 +1,86 @@
+import { Category, DealWithTags } from '@/types/Types'
+import React from 'react'
+import DealImage from '../DealImage'
+import TagsList from '../../forms/add-a-deal/TagsList'
+import ClickableCoupon from '../../ClickableCouponCode'
+import { format } from 'date-fns'
+import Image from 'next/image'
+import Section, { SECTION_STYLE, SECTION_WIDTH } from '../../Section'
+import ShareDealButton from './ShareDealButton'
+import ReportDealButton from './ReportDealButton'
+
+interface DealDetailsProps {
+  deal: DealWithTags
+}
+export default function DealDetails({ deal }: DealDetailsProps) {
+  return (
+    <div className="text-white">
+      <Section width={SECTION_WIDTH.SM} className="pb-32 sm:pb-40 md:pb-80">
+        <div className="mb-10">
+          <p className="mb-2 font-bold  text-teal-500">{deal.category || ''}</p>
+          <div className="mb-10 flex flex-row items-end justify-between gap-x-4">
+            <h1 className="inline text-2xl transition-colors hover:text-teal-500 md:text-3xl">
+              {deal.name}
+            </h1>
+            <a
+              href={`/api/tracking?id=${deal.xata_id}&url=${deal.link}`}
+              className="flex cursor-pointer items-center gap-x-2 rounded-md bg-teal-500 px-2 py-1 text-black transition-all hover:scale-105 hover:bg-teal-500/90"
+            >
+              <span>Visit</span>
+              <Image
+                src="/icons/upright-arrow.svg"
+                alt="arrow right"
```

Comment:
I feel like this could be an empty alt text since it's merely decorative and you already have the span of "Visit" there. @EmmaDawsonDev, your thoughts? A11y is obviously more your wheelhouse than mine.

Reply:
Thanks! Updated to use empty alt